src/components/SelectionList/CategorySelectorListItem.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {useState} from 'react';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CategorySelector from '@pages/workspace/distanceRates/CategorySelector';
+import * as Policy from '@userActions/Policy/Policy';
+import BaseListItem from './BaseListItem';
+import type {InviteMemberListItemProps, ListItem} from './types';
+
+function CategorySelectorListItem<TItem extends ListItem>({item, onSelectRow, isFocused}: InviteMemberListItemProps<TItem>) {
```

Comment:
I think type `InviteMemberListItemProps` here is by falsely, we can use  `BaseListItemProps` or create `categoryListItemProps` if needed